pom.xml

```diff
@@ -282,6 +283,49 @@
                         <link>http://fasterxml.github.io/jackson-databind/javadoc/2.6/</link>
                         <link>https://javaee-spec.java.net/nonav/javadocs/</link>
                     </links>
+                    <!-- optional dependencies from other modules (must be explicitly declared here in order to be correctly resolved) -->
+                    <additionalDependencies>
```

Comment:
For my own benefit, what difference does moving this up make?  Is it just so that all the modules can benefit from these instead of just core?